### PR TITLE
fix(planet-minecraft): buttons

### PR DESCRIPTION
**Pull request by somerand0mcat**
## 🔧 What does this fix? 🔧
as said in the title
image
image

## 🗒 Checklist 🗒

- [X] I have read and followed Catppuccin's [contributing guidelines](https://github.com/catppuccin/userstyles/blob/main/docs/CONTRIBUTING.md).
- [X] I have updated the version appropriately in the `==UserStyle==` header of the `catppuccin.user.css` file.

### CODE REVIEW

**Comment by uncenter** [comment]
Looks like you got rid of the hover effect? In the original screenshot there are two shades of blue but in your new one it is a single color (mantle).

File: styles/planet-minecraft/catppuccin.user.css
Comment (uncenter) [inline styles/planet-minecraft/catppuccin.user.css:139]: There are tons of tohher `.content-actions` sections in this userstyle, some of which appear to have been for theming this buttons in the past. Can you consolidate and nest the `.content-actions` sections into one?

**Review by uncenter — CHANGES_REQUESTED** [review]

File: styles/planet-minecraft/catppuccin.user.css
Comment (somerand0mcat) [reply styles/planet-minecraft/catppuccin.user.css:139]: sure

File: styles/planet-minecraft/catppuccin.user.css
Comment (uncenter) [inline styles/planet-minecraft/catppuccin.user.css:367]: There are more lines above this line here (starting at line 340) that are `.content-actions` related.

**Review by uncenter — CHANGES_REQUESTED** [review]

**Comment by somerand0mcat** [comment]
i accidentally referenced #2 on that commit, whoops

**Comment by somerand0mcat** [comment]
> Looks like you got rid of the hover effect? In the original screenshot there are two shades of blue but in your new one it is a single color (mantle).

i guess i'll change it to ``@base``

File: styles/planet-minecraft/catppuccin.user.css
Comment (uncenter) [inline styles/planet-minecraft/catppuccin.user.css:148]: Nesting can be improved here, and let's stick with `surface0` for these buttons rather than `mantle` as it was before.

**Review by uncenter — CHANGES_REQUESTED** [review]

**Comment by somerand0mcat** [comment]
is this PR going to be merged anytime soon?

**Comment by uncenter** [comment]
Any reason?

**Comment by somerand0mcat** [comment]
> Any reason?

the pr's been open for 3 weeks and i gave up on waiting for the maintainer to merge it

File: styles/planet-minecraft/catppuccin.user.css
Comment (uncenter) [inline styles/planet-minecraft/catppuccin.user.css:147]: Most times with nested CSS the lines for this selector go first, and then the nested stuff.
```suggestion
              background: @surface0 !important;
              border-color: @surface0 !important;

              a {
                  color: @text !important;
              }
              
              a:hover, .js_link:hover, .link:hover {
                  background-color: @surface2 !important;
                  color: @text !important;
              }
              
              .material-icons {
                  color: @subtext1 !important;
              }
```

**Review by uncenter — CHANGES_REQUESTED** [review]

File: styles/planet-minecraft/catppuccin.user.css
Comment (uncenter) [inline styles/planet-minecraft/catppuccin.user.css:146]: This wasn't the original color, and you missed the part where if any of the links above are hovered the icon changes to accent.

File: styles/planet-minecraft/catppuccin.user.css
Comment (uncenter) [inline styles/planet-minecraft/catppuccin.user.css:149]:
```suggestion
        a,
        .link,
        .js_link,
        .js_link_m {
          color: @text !important;
          
          &:hover {
          	color: @text !important;
          	background-color: @surface1 !important;
          	
          	.material-icons {
          		color: @accent-color !important;
          	}
          }
        }
```

File: styles/planet-minecraft/catppuccin.user.css
Comment (uncenter) [inline styles/planet-minecraft/catppuccin.user.css:152]:
```suggestion
          color: @subtext0 !important;
```
Please stop changing the colors. It was `subtext0` originally and I don't see a reason to change it.

**Review by uncenter — CHANGES_REQUESTED** [review]

File: styles/planet-minecraft/catppuccin.user.css
Comment (somerand0mcat) [reply styles/planet-minecraft/catppuccin.user.css]: i accidentally changed that while testing the css, sorry

**Review by uncenter — CHANGES_REQUESTED** [review]
Please also remove the old latte only stuff (lines 167-192).

File: styles/planet-minecraft/catppuccin.user.css
Comment (uncenter) [inline styles/planet-minecraft/catppuccin.user.css:165]: @nyameliaaaa do you remember where this stuff is? I'm a bit confused by the comment above and why this is necessary to be only for light mode. There might be a better solution or maybe we don't need this at all.

Comment (somerand0mcat) [reply styles/planet-minecraft/catppuccin.user.css:165]: i think you'll have to contact her on discord since shes rarely on catppuccin's github

**Pull request closed without merging**
